doc/source/reference/simd/how-it-works.rst

```diff
@@ -347,3 +347,21 @@ through ``--cpu-dispatch``, but it can also represent other options such as:
           // #include "hello2.dispatch.h"
           // DISPATCH_CALL_HIGH(another_function, ("the highest interest"))
       }
+
+
+.. _runtime-simd-dispatch::
+
+**********************************************
+How does the CPU dispatcher work (run time)?
+**********************************************
+
+Importing NumPy triggers a scan of the available CPU features from the set
+of dispatchable features. This can be further restricted by setting the
+environment variable ``NPY_DISABLE_CPU_FEATURES`` to a comma-, tab-, or
+space-separated list of features to disable. This will raise an error if
+parsing fails or if the feature was not enabled. For instance, on ``x86_64``
+this will disable ``AVX2`` and ``FMA3``::
+
+    NPY_DISABLE_CPU_FEATURES="AVX2,FMA3"
+
```

Comment:
Better to move it to https://github.com/numpy/numpy/blob/main/doc/source/reference/simd/build-options.rst#runtime-trace. This document will be removed soon or later after moving to meson.

Reply:
done